### DISCUSSION TITLE
Recharge Question

### DISCUSSION THREAD

**Discussion by coder0103** [pirateIV/experiments#6] [Q&A] [answered]
How do you make pin generate

**Comment by pirateIV** [comment, marked as answer]
use a random number to generate 11 pin numbers check the js well...

**Reply by coder0103** [reply]
Thanks

**Comment by coder0103** [comment]
`let pin = Math.floor(Math.random() * 11)
return pin
`

**Reply by coder0103** [reply]
Generates 11 random numbers consisting of `0 - 9` numver @ben64103

**Reply by pirateIV** [reply]
`function generateRandom(){`
 ` let pin = Math.floor(Math.random() * 11)`

   `return pin`
`}`
